assets/src/edit-story/elements/text/edit.js

```diff
@@ -232,6 +239,14 @@ function TextEdit({
     ]);
   }, [font, fontFaceSetConfigs, maybeEnqueueFontStyle]);
 
+  useTransformHandler(id, (transform) => {
+    const target = textBoxRef.current;
+    const updatedFontSize = transform?.updates?.fontSize;
```

Comment:
It might be better for this component to compute the `fontSize` itself, and possibly to turn off that feature in the `SingleSelectionMovable`.

Reply:
Curious to know the reasoning.
Currently, it's exactly matching the font size in the display -- tested out two equal elements when resizing in display mode vs edit mode.

Also, if the font calculation would be done separately here, wouldn't the history entries include mismatching data? E.g. the height/width/left/top would be set via Moveable to a history entry but the font-size would be added as it was before resizing since it wouldn't be calculated and saved together with the size and position. This would include a history entry with font size out of sync compared to the other props. Thoughts?